backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlDatabaseFieldFactory.java

```diff
@@ -19,6 +19,51 @@
 
 public class GraphqlDatabaseFieldFactory {
 
+  public static final GraphQLObjectType permissionsMetadataType =
+      new GraphQLObjectType.Builder()
+          .name("MolgenisPermissionsType")
+          .field(
+              GraphQLFieldDefinition.newFieldDefinition()
+                  .name(TABLE_SCHEMA)
+                  .type(Scalars.GraphQLString))
+          .field(
+              GraphQLFieldDefinition.newFieldDefinition()
+                  .name(TABLE_NAME)
+                  .type(Scalars.GraphQLString))
+          .field(
+              GraphQLFieldDefinition.newFieldDefinition()
+                  .name(GROUP_NAME)
+                  .type(Scalars.GraphQLString))
+          .field(
+              GraphQLFieldDefinition.newFieldDefinition()
```

Comment:
I am a bit confused that users + groups are here too. Seems orthogonal freature. Don't we have a group mgt function for this already?